install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -425,6 +425,10 @@ data:
   enable-local-redirect-policy: {{ .Values.localRedirectPolicy | quote }}
 {{- end }}
 
+{{- if hasKey .Values.bpf "socketLBHostnsOnly" }}
```

Comment:
I would move this similarly under `.Values.loadBalancer.HostNamespaceOnly`

Reply:
Done.